app/controllers/reviews_controller.rb

```diff
@@ -24,7 +24,7 @@ def update
     @review = Review.find(params[:id])
 
     if current_user == @review.user
-      if @review.update(review_params)
+      if @review.update(review_params.merge(edited: true))
```

Comment:
Instead of saving a flag that tells us if this review has been edited, we can compare the `created_at` and `updated_at` fields of the review.

Write an instance method on the review model called `edited?` that does this comparison and returns true or false.